examples/reuploading_classifier/main.py

```diff
@@ -31,7 +36,11 @@ def main(dataset, layers):
         print('Problem never solved, finding optimal parameters...')
         result, parameters = ql.minimize(
             method='l-bfgs-b', options={'disp': True})
-        data[dataset][layers] = parameters
+        try:
+            data[dataset][layers] = parameters
+        except:
+            data[dataset] = {}
```

Comment:
I think you can avoid this by making sure e.g. in line 30 that `data[dataset]` is not empty / not defined.

Reply:
Indeed